### PR TITLE
Added No Autocomplete version of searchbar

### PR DESCRIPTION
**Pull request by uoa-lchu090**
- made it conditional with props
- outputs filtered data when no autocomplete

### CODE REVIEW

File: forgettable-frontend/src/Components/SearchBar/SearchBar.js [hunk 1]
```diff
@@ -22,48 +22,77 @@ function SearchBar({placeholder, data}) {
     } else {
       setFilteredData(newFilter);
     }
+
+    if (hasAutocomplete === false){
+      // outputs data so that pages can use the filtered data
+    exportFilteredData(filteredData);
```
Comment (uoa-lchu090) [inline forgettable-frontend/src/Components/SearchBar/SearchBar.js:28]: I think for this one to be successfully output you need to add in:
 const [data, setData] = useState('');
  
  const exportFilteredData = (filteredData) => {
    setData(filteredData);
  }

then when declaring the SearchBar component you need to add exportFilteredData={exportFilteredData}

**Review by rainasong — APPROVED** [review]
We might need to discuss how to do filter for data without 'title' field later but currently looks good to me!

**Review by dkap937-209 — APPROVED** [review]
LGTM

**Review by noahjade — APPROVED** [review]
lgtm!

**Pull request merged**